homeassistant/components/heos/__init__.py

```diff
@@ -0,0 +1,51 @@
+"""Denon HEOS Media Player."""
+
+import asyncio
+import logging
+import voluptuous as vol
```

Comment:
Please add a blank line between standard library and 3rd party import groups.

Reply:
ok